src/main/webapp/js/instructorFeedbackEdit/questionRubric.es6

```diff
@@ -118,12 +109,51 @@ function addRubricCol(questionNum) {
         $(rubricRowFragment).insertAfter(lastTd);
     }
 
+    // Add options row at the end
+    const optionsRow =
+        `<td class="align-center rubricCol-${questionNum}-${newColNumber - 1}" data-col="${newColNumber - 1}">
+            <div class="btn-group">
+                <button type="button" class="btn btn-default" id="rubric-move-col-left-${questionNum}-${newColNumber - 1}"
+                        onclick="swapRubricCol(${newColNumber - 1}, ${questionNum}, true)" data-toggle="tooltip"
+                        data-placement="top" title="Move column left">
+                    <span class="glyphicon glyphicon-arrow-left"></span>
+                </button>
+                <button type="button" class="btn btn-default" id="rubricRemoveChoiceLink-${questionNum}-${newColNumber - 1}"
+                        onclick="removeRubricCol(${newColNumber - 1}, ${questionNum})"
+                        onmouseover="highlightRubricCol(${newColNumber - 1}, ${questionNum}, true)"
+                        onmouseout="highlightRubricCol(${newColNumber - 1}, ${questionNum}, false)">
+                    <span class="glyphicon glyphicon-remove"></span>
+                </button>
+                <button type="button" class="btn btn-default" id="rubric-move-col-right-${questionNum}-${newColNumber - 1}"
+                        onclick="swapRubricCol(${newColNumber - 1}, ${questionNum}, false)" data-toggle="tooltip"
+                        data-placement="top" title="Move column right">
+                    <span class="glyphicon glyphicon-arrow-right"></span>
+                </button>
+            </div>
+        </td>`;
+
+    const lastTd = $(`#rubric-options-row-${questionNum} td:last`);
+    $(optionsRow).insertAfter(lastTd);
+
+    // Initialize tooltips
+    $(`#rubric-move-col-left-${questionNum}-${newColNumber - 1}`).tooltip({ container: 'body' });
+    $(`#rubric-move-col-left-${questionNum}-${newColNumber - 1}`).click(() => {
+        swapRubricCol(questionNum, newColNumber - 1, true);
+    });
+
+    $(`#rubric-move-col-right-${questionNum}-${newColNumber - 1}`).tooltip({ container: 'body' });
+    $(`#rubric-move-col-right-${questionNum}-${newColNumber - 1}`).click(() => {
```

Comment:
Consider extracting the left and right buttons as variables first. Remember to use the $ prefix if necessary.